### ISSUE TITLE
Remove the Fire Immunity attribute overhead icon

### ISSUE THREAD

**Issue by EnderdragonLord** [Tslat/Advent-Of-Ascension#1768]
Reason I'd like to suggest this is because I don't see a point of having it around when vanilla mobs (and modded mobs) most notably Zombie Pigmen already are immune to fire but don't have the overhead icon itself. I think that it just makes things too inconsistent and confusing and that it would be best to remove the overhead icons completely from all the mobs that have it but still keeping their fire immunity.

**Comment by Cruuk** [comment]
To be honest I disagree. Why does vanilla doing a thing stop us from doing another thing? It's just removing practicality for the sake of "consistency", and I don't like that. Gingerbread man and Gingerbird are immune to fire, how would a player know that they're fire immune without doing prior research?

If it was removed, I can see easily people also questioning "Why do all forms of damage have an immunity icon except fire? It's inconsistent."

The Wither becomes immune to ranged damage at half-HP, but doesn't use the Ranged immune icon added by Advent. Should the Ranged immunity icon be removed too. I'd say it's pretty important to know when a mob is immune to ranged damage. I just don't agree with obfuscating information like this.

**Comment by EnderdragonLord** [comment]
I can totally understand what you mean but I wasn't asking for everything to be the same as vanilla. 

It's mostly about how the fire immunity icon just seems unnecessary when there's pre-existing vanilla and potential third party mobs having fire immunity already without any sort of indication unlike AoA mobs, I'd say it's just as confusing when that's a current thing "Why is this mob immune to fire but it doesn't sport the same icon? What if the icon stands for something else?" It could be fixed by somehow adding icons to those mobs and I'm not a mod maker, I've tinkered with code, and I know that stuff is difficult to do. 

I figured it would be better to remove it, some people could be confused by this but it was a thing in Minecraft for years and lots of players would already know that some mobs are completely immune to fire without indication.

It's true that there exists mobs immune to Ranged damage already, but I'm pretty sure the Wither had a shield at half health, players would discover that it's immune to projectiles when they notice how the shield affects those arrows, Endermen also teleport when hit by arrows, and in my opinion I think it's best for mobs to have something like that rather than an icon to not make it too obvious on what it's immune to and for there to be some interesting mechanics to discover from each mob.
